1.intro/chapter3/customize/notebooks-customize.ipynb

```diff
@@ -0,0 +1,1068 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Customizing your Workbench with Pip\n",
+    "\n",
+    "You can use the package installer for Python (`pip`) to install Python packages.\n",
+    "With `pip`, you can customize your workbench, by installing libraries (or different versions of those libraries) that are initially not included in the workbench.\n",
+    "\n",
+    "> **NOTE**\n",
+    ">\n",
+    "> You can also use other package managers such as Conda.\n",
```

Comment:
Is Conda available in Red Hat shipped images?

Reply:
It can be enabled by enabling the Anaconda SaaS integration https://access.redhat.com/documentation/es-es/red_hat_openshift_data_science/1/html-single/getting_started_with_red_hat_openshift_data_science/index#enabling-services_get-started